frontend/src/services/effects.ts

```diff
@@ -255,8 +255,15 @@ export function withEffects(store: Store) {
     .on('adequacies')
     .filter(Boolean)
     .subscribe(adequacies => {
-      const representativePoints = store.get('representativePoints')
-      store.set('pointGeoJson')(representativePointsToGeoJSON(adequacies)(representativePoints))
+      let representativePoints = store.get('representativePoints')
+      if (representativePoints === null) {
+        store.set('pointFeatureCollections')(null)
+      } else {
+        let nChunks = Math.floor(representativePoints.length / 10)
```

Comment:
This is a slightly misleading name. The number of chunks is 10 here - what this variable actually represents is the chunk size.